src/anm/media/audio.js

```diff
@@ -47,7 +47,7 @@ var audioContext = getAudioContext();
  * @class anm.Audio
  */
 function Audio(url) {
-    this.url = url + audioExt;
+    this.url = /\.\w+$/i.test(url) ? url : url + audioExt;
```

Comment:
@RobertoUa Please note that when we add audio source, `audioType` is also required. Here it is passed to a function: https://github.com/Animatron/player/pull/392/files#diff-4e959658da38c66435f3dd42b062b504L183 , and here it is constructed: https://github.com/Animatron/player/pull/391/files#diff-4e959658da38c66435f3dd42b062b504R14.